modules/consensus/src/lib.rs

```diff
@@ -0,0 +1,172 @@
+use futures::channel::mpsc::SendError as FuturesSendError;
+use futures::Sink;
+use futures::SinkExt;
+use futures_util::StreamExt;
+use solana_lite_rpc_core::block_store::BlockInformation;
+use solana_sdk::epoch_info::EpochInfo;
+use solana_transaction_status::EncodedConfirmedBlock;
+use std::collections::HashMap;
+use thiserror::Error;
+use tokio::pin;
+use tokio::task::JoinHandle;
+use yellowstone_grpc_client::GeyserGrpcClient;
```

Comment:
IMO we should move technology/infrastructure code for getting/sending data out of the domain modules into infrastructure-modules (e.g. infrastructure/geyser-subscription-yellowstone)